### PR TITLE
fix(ci): Scanner V4 Breakage in Existing Tests

### PR DESCRIPTION
**Pull request by dcaravel**
## Description

~Skip / adjust tests that will break when Scanner V4 enabled (for now) to bring CI back to healthy state.~

Disables Scanner V4 in CI tests that use the ACS operator.

ROX-22088 has been created to revert these changes in future when Scanner V4 available in CI

## Checklist
- [ ] Investigated and inspected CI test results
- [x] ~Unit test and regression tests added~
- [x] ~Evaluated and added CHANGELOG entry if required~
- [x] ~Determined and documented upgrade steps~
- [x] ~Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))~

If any of these don't apply, please comment below.

## Testing Performed

CI

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: qa-tests-backend/src/main/groovy/services/FeatureFlagService.groovy [hunk 1]
```diff
@@ -19,4 +19,12 @@ class FeatureFlagService extends BaseService {
         }
         return flag.enabled
     }
+
+    static boolean isFeatureFlagEnabled(String envVar, boolean defaultVal) {
```
Comment (janisz) [inline qa-tests-backend/src/main/groovy/services/FeatureFlagService.groovy:23]: How about making it just for scanner? We used to have something similar for PG migration
`isScannerV4()

**Comment by janisz** [comment]
>  revert the skips in future when Scanner V4 available in CI
why it's not available in CI now?

**Comment by dcaravel** [comment]
> > revert the skips in future when Scanner V4 available in CI

> why it's not available in CI now?

TL;DR; believe because vulns take a long time to load and likely will be unavail in CI timeframe.  

The 'matcher' component does not start with an initial set of vulns (unlike the existing stackrox scanner), in some tests we've seen it take as much as 6 hours to load the initial data - as it stands right now the matcher will return a 'successful' scan with zero vulns if the data hasn't loaded yet (being worked on) - which is what I believe is causing the errors in the tests that expect vulns in the scan results.

File: qa-tests-backend/src/main/groovy/services/FeatureFlagService.groovy (hunk 1 above)
Comment (dcaravel) [reply qa-tests-backend/src/main/groovy/services/FeatureFlagService.groovy:23]: Great suggestion, that is better.  May be removing this all together in the short term as am currently leaning toward disabling Scanner V4 in CI until we figure this out the setup more holistically - should we keep this logic, will make the update.  Thanks @janisz

**Comment by dcaravel** [comment]
> test ocp-4-14-qa-e2e-tests

Failure is unrelated to revious failures `Post failure in: ['scripts/ci/collect-collector-metrics.sh', 'stackrox', '/tmp/collector-metrics']`

**Review by gavin-stackrox — APPROVED** [review]
LGTM the test failures are different flakes
